lib/qx/tool/cli/Cli.js

```diff
@@ -0,0 +1,110 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2018 Zenesis Ltd
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+     * John Spackman (john.spackman@zenesis.com, @johnspackman)
+
+************************************************************************ */
+
+require("qooxdoo");
+const path = require("path");
+
+/**
+ * Entry point for the CLI
+ */
+
+qx.Class.define("qx.tool.cli.Cli", {
+  extend: qx.core.Object,
+  
+  members: {
+    run: function() {
+      // sanitise process list parameter
+      let args = [path.relative(process.cwd(), process.argv[1])]
+
+      for (let i=2; i<process.argv.length; i++) {
+        let arg = process.argv[i]
+        if (arg === '--token' || arg === '-t') {
```

Comment:
The token parameter has been removed, so this is no longer necessary, is it?

Reply:
no it's not :)  I'll remove it